ovn-fake-multinode-utils/playbooks/install-dependencies-deb.yml

```diff
@@ -0,0 +1,7 @@
+- name: Install dependencies
```

Comment:
As mentioned above, I think we can merge this back into a single `install-dependencies.yml` where we add two conditional tasks (depending on OS family).  Wdyt?

Reply:
Done